assembl/static2/js/app/components/administration/voteSession/modulesSection.jsx

```diff
@@ -1,22 +1,43 @@
 // @flow
 import React from 'react';
-import { I18n } from 'react-redux-i18n';
+import { I18n, Translate } from 'react-redux-i18n';
 import { connect } from 'react-redux';
-import { Checkbox } from 'react-bootstrap';
+import { Checkbox, SplitButton, MenuItem } from 'react-bootstrap';
+import range from 'lodash/range';
 import SectionTitle from '../sectionTitle';
 import Helper from '../../common/helper';
 import TokensForm from './tokensForm';
-import { createTokenVoteModule, deleteTokenVoteModule } from '../../../actions/adminActions/voteSession';
+import GaugeForm from './gaugeForm';
+import {
+  createTokenVoteModule,
+  deleteTokenVoteModule,
+  createGaugeVoteModule,
+  deleteGaugeVoteModule
+} from '../../../actions/adminActions/voteSession';
 
 type ModulesSectionProps = {
   tokenModules: Object,
+  gaugeModules: Object,
```

Comment:
same as above, try to avoid `object` type (it tells almost nothing and will not help us to detect errors in the code)